docs/changelog.md

```
@@ -5,6 +5,15 @@ description: The latest updates and changes to CodeRabbit.
 sidebar_position: 9
 ---
 
+##  Security and SAST Remediation Improvements
+#### Oct 22, 2024
```

Markdownlint

Expected: h3; Actual: h4
Heading levels should only increment by one level at a time

(MD001, heading-increment)